src/app/designs/page.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import Card from '@/Components/Card';
+import Image from 'next/image';
+
+interface Product {
+    id: number;
+    imageUrl: string;
+    title: string;
+    price: string;
+    rating: string;
+}
+
+const products: Product[] = [
+    {
+        id: 1,
+        imageUrl: '/design1.png',
+        title: 'Spread Collar Shirt',
+        price: '48.99',
+        rating: '5.0',
+    },
+    {
+        id: 2,
+        imageUrl: '/design2.png',
+        title: 'White Solid Formal Shirt',
+        price: '39.0',
+        rating: '4.9',
+    },
+    {
+        id: 3,
+        imageUrl: '/design3.png',
+        title: 'Gray Solid Padded Jacket',
+        price: '39.0',
+        rating: '4.8',
+    },
+    {
+        id: 4,
+        imageUrl: '/design4.png',
+        title: 'Printed Loose T-shirt',
+        price: '39.99',
+        rating: '4.7',
+    },
+    {
+        id: 1,
+        imageUrl: '/design1.png',
+        title: 'Spread Collar Shirt',
+        price: '48.99',
+        rating: '5.0',
+    },
+    {
+        id: 2,
+        imageUrl: '/design2.png',
+        title: 'White Solid Formal Shirt',
+        price: '39.0',
+        rating: '4.9',
+    },
+    {
+        id: 3,
+        imageUrl: '/design3.png',
+        title: 'Gray Solid Padded Jacket',
+        price: '39.0',
+        rating: '4.8',
+    },
+    {
+        id: 4,
+        imageUrl: '/design4.png',
+        title: 'Printed Loose T-shirt',
+        price: '39.99',
+        rating: '4.7',
+    },
+];
+
+const Home = () => {
```

Comment:
Kindly change the name of the function to something more descriptive. The name Home is clashing with Home function from the main page.tsx. This is causing a stackoverflow error when we try to render it (it is recursing to itself). Perhaps a more informational name like DesignPage, DesignWindow etc. This will also help with the development process in the team.

Reply:
renamed to DesignsSection